client/data/account-keys/actions.js

```diff
@@ -0,0 +1,55 @@
+import { dispatch, select } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { apiFetch } from '@wordpress/data-controls';
+import { NAMESPACE, STORE_NAME } from '../constants';
+import ACTION_TYPES from './action-types';
+
+export function updateAccountKeysValues( payload ) {
+	return {
+		type: ACTION_TYPES.SET_ACCOUNT_KEYS_VALUES,
+		payload,
+	};
+}
+
+export function updateAccountKeys( payload ) {
```

Comment:
QQ: Why do we need 2 functions `updateAccountKeysValues` and `updateAccountKeys`? 

For example we have `publishable_key`/`secret_key`. We are supposed to update their values only right? Isn't `updateAccountKeysValues` enough for this?

Reply:
`updateAccountKeysValues` updates one (or more) values in the store, `updateAccountKeys` is instead used as something that wipes all the data in the store and replaces it with its payload.

`updateAccountKeysValues` is meant to be used to update one single value (e.g.: from a component), whereas `updateAccountKeys` is meant to be used for updating _all_ the values when the data is fetched from the backend (so that any possible changes are reverted).